exporters/cht-user-management/scrape.yml

```
@@ -0,0 +1,7 @@
+scrape_configs:
+  - job_name: cht-user-management-metrics
+    metrics_path: /metrics
+    scheme: "https"
+    static_configs:
+      - targets:
+          - users-chis-ke.app.medicmobile.org
```

Our other configs leave this as a [dummy value](https://github.com/medic/cht-watchdog/blob/d8aa62ed99cf05dfca336ec053f8f9d3fd97113e/exporters/postgres/sql_servers_example.yml#L31) like `user-man-tool-url.example.org` or something

don't feel too strongly about removing this, but light suggestion we change it:

```suggestion
          - user-man-tool-url.example.org
```

Then I the docs we publish about how to deploy, we'll call out that you need to put the URL(s) of your instances in here.